### PR TITLE
Update language on main community page 

### PR DESCRIPTION
**Pull request by danielle06**
updated some of the verbiage for digestibility

### CODE REVIEW

**Comment by maneesha** [comment]
Thanks @danielle06 
Assigning to @OscarSiba  to review for language

File: content/community/_index.md [hunk 1]
```diff
@@ -9,18 +9,18 @@ cascade:
 
 The Carpentries works to help institutions and individuals spread skills for data analysis, computational thinking, and research software development through building local and global communities of practice.
 
-We remain a nearly all-volunteer organisation. Our work is made possible by the efforts of our amazing volunteers, workshop hosts and organisers, community champions, member organisations, and Core Team (paid staff).
+We continue to be an organization primarily supported by volunteers. Our work is made possible through the efforts of our amazing workshop hosts, organizers, community champions, member organizations, and our paid staff, the Core Team.
```
Comment (OscarSiba) [inline content/community/_index.md:12]:
```suggestion
We continue to be an organisation primarily supported by volunteers. Our work is made possible through the efforts of our amazing workshop hosts, organisers, community champions, member organisations, and our paid staff, the Core Team.
```

File: content/community/_index.md [hunk 2]
```diff
@@ -9,18 +9,18 @@ cascade:
...
-The Carpentries is a community of practice made up by various regional and role-related communities. Below is a list of our role-related volunteer communities. Select a volunteer community to learn more.
+The Carpentries is a community of practice composed of regional and role-related volunteer communities. Below is a list of our role-related volunteer communities. Select a community to learn more about their role at The Carpentries.
 
-- [Curriculum advisors](/community/curriculum_advisors/) - those who guide development of existing - lessons
-- [Code of Conduct Committee](/community/coc_ctte) - those who keep our community friendly - and welcoming
-- [Executive Council](/about-us/governance/) - community members that constitute the - governance and policy arm of The Carpentries
-- [Instructors](/community/instructors) - those who teach and sometimes organise workshops
-- [Lesson maintainers](/community/maintainers) - those who support existing lessons
-- [Instructor Trainers](/community/instructor-trainers) - those who prepare our new Instructors
+- [Curriculum advisors](/community/curriculum_advisors/) - guide the development of existing - lessons
```
Comment (OscarSiba) [inline content/community/_index.md:18]:
```suggestion
- [Curriculum advisors](/community/curriculum_advisors/) - guide the development of existing lessons
```

File: content/community/_index.md [hunk 3]
```diff
@@ -9,18 +9,18 @@ cascade:
...
+- [Code of Conduct Committee](/community/coc_ctte) - keep our community friendly - and welcoming
```
Comment (OscarSiba) [inline content/community/_index.md:19]:
```suggestion
- [Code of Conduct Committee](/community/coc_ctte) - keep our community friendly and welcoming
```

File: content/community/_index.md [hunk 4]
```diff
@@ -9,18 +9,18 @@ cascade:
...
+- [Executive Council](/about-us/governance/) - make up the governance and policy portion of The Carpentries
```
Comment (OscarSiba) [inline content/community/_index.md:20]:
```suggestion
- [Advisory Committee and Board of Directors](/about-us/governance/) - make up the governance and policy portion of The Carpentries
```

File: content/community/_index.md [hunk 5]
```diff
@@ -9,18 +9,18 @@ cascade:
...
+- [Instructors](/community/instructors) - teach and sometimes organise workshops
+- [Lesson maintainers](/community/maintainers) - support existing lessons
```
Comment (OscarSiba) [inline content/community/_index.md:22]:
```suggestion
- [Lesson Maintainers](/community/maintainers) - support existing lessons
```

File: content/community/_index.md [hunk 6]
```diff
@@ -9,18 +9,18 @@ cascade:
...
+- [Instructor Trainers](/community/instructor-trainers) - prepare our new Instructors
```
Comment (OscarSiba) [inline content/community/_index.md:23]:
```suggestion
- [Instructor Trainers](/community/instructor-trainers) - train and support our new Instructors
```

**Review by OscarSiba — COMMENTED** [review]
Not sure @maneesha if you'd like to first look through these before approving, otherwise here they are.

**Pull request merged**